build_projects/update-dependencies/Program.cs

```diff
@@ -60,6 +60,20 @@ public static void Main(string[] args)
             }
         }
 
+        private static BuildInfo GetBuildInfo(string name, string buildInfoFragment, bool fetchLatestReleaseFile = true)
+        {
+            const string FileUrlProtocol = "file://";
```

Comment:
`BuildInfo.Get` calls into HttpClient, which (at least on *nix) does not accept the `file://` protocol (just `http://` and `https://`) so we do this here.